source/channels/send-messages.rst

```diff
@@ -65,6 +65,11 @@ Select **Edit** to edit your own messages. Editing a message won't trigger new @
 
 Select **Delete** to delete your own messages. Select **Delete** again to confirm.
 
+From Mattermost v7.9, you can view the edit history of an edited message and restore a previous version of an edited message.
```

Comment:
> From Mattermost v7.9

Do we need to state this?

Reply:
We usually do if the change doesn't apply to previous versions and the customer needs to upgrade to use it.

Reply:
@esethna One common piece of feedback we receive from docs site visitors is that a given feature isn't available to them (because they're on an older version of Mattermost).

One clear way forward is to set expectations up front about the minimum release needed to access a feature. Another, longer term approach could be versioned documentation.